<?xml version="1.0"?>
<launch>

  <arg name="simulated_robot" default="true" />
  <arg name="odom_topic_name" default="odom" />
  <arg name="auto-close" default="false" doc="Startup mode" />
  
  <param name="/use_sim_time" value="true" />


  <node name="scout_webots_node" pkg="scout_webots_sim" type="scout_webots_node" required="$(arg auto-close)" output="screen">
    <param name="use_sim_time" value="true" />
    <param name="clock" value="/clock" />
    <param name="odom_frame" type="string" value="odom" />
    <param name="odom_topic_name" type="string" value="$(arg odom_topic_name)" />
    <param name="base_frame" type="string" value="base_link" />
    <param name="simulated_robot" type="bool" value="true" />
    <param name="sim_control_rate" value="50" />
  </node>
</launch>
